doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -19,29 +19,34 @@
 
 ## Executive Summary
 
-The Reference Conformance 2 (RC2) was established to ensure implementations of the Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet industry driven quality assurance standards for conformance, verification, and validation. For the purpose of this chapter, infrastructure and cloud native network function (CNF) testing will be performed to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of software delivery.  More specifically, Conformance includes:
+The RC2 was established to ensure implementations of the CNTT Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet functional and performance requirements derived from RA2 and the CNTT Reference Model (RM). For the purpose of this chapter, cloud infrastructure and cloud native network function (CNF) testing will be specified to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of hardware and software delivery that is within the scope of CNTT. Conformance scope includes:
```

Comment:
```suggestion
The Reference Conformance for the Kubernetes-based workstream (RC2) was established to ensure implementations of the CNTT Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet functional and performance requirements specified in RA2 and the CNTT Reference Model (RM). Cloud infrastructure and cloud native network function (CNF) verification and validation will be utilised to evaluate **Conformance** (i.e. adherence) to the Ra-2 and RM specifications. Conformance scope includes:
```

Reply:
Justification
1. I think Conformance is the theme of RC-2 and not just Chapter 1
2. proficiency -- "a high degree of competence or skill; expertise" -- is inapplicable